tasks/scripts/k8s-topgun

```diff
@@ -24,4 +24,4 @@ go mod download
 
 go install github.com/onsi/ginkgo/ginkgo
 
-ginkgo -nodes=4 -race -keepGoing -slowSpecThreshold=300 -skip="$SKIP" ./topgun/k8s/ "$@"
+ginkgo -nodes=8 -race -keepGoing -slowSpecThreshold=900 -flakeAttempts=3 -skip="$SKIP" ./topgun/k8s/ "$@"
```

Comment:
`-flakeAttempts`: https://github.com/onsi/ginkgo/pull/261

`-nodes=8` (double what we used before) is not possible due to the use of more conservative `resources.requests` (see PR that made this possible).

Reply:
flakeAttempts has a lively conversation around it onsi/ginkgo#260!

i do agree with some of the ideas expressed in that issue, would `flakeAttempts` be necessary with running topgun inside the k8s cluster? just wondering how stable k8s-topgun gets with the new setup...

Reply:
running it dozens of times showed that one time or another, there's a flake still 😬  I'd say we can keep track of that with a story around capturing those results like they said in the `ginkgo` issue (to keep track of the failures)

wdyt?

Reply:
oh, and there's also the fact that we optimistically wait autoscaling to trigger on time, which can flake too

Reply:
you've thought about this, excellent! a story about capturing those results somehow would be pretty nifty. i'm convinced!